lib/std/crypto/kyber.zig

```diff
@@ -189,35 +186,37 @@ fn Kyber(comptime p: Params) type {
         /// Algorithm name.
         pub const name = p.name;
 
+        /// A shared secret, and an encapsulated (encrypted) representation of it.
+        pub const EncapsulatedSecret = struct {
+            shared_secret: [shared_length]u8,
+            ciphertext: [ciphertext_length]u8,
+        };
+
         /// A Kyber public key.
         pub const PublicKey = struct {
             pk: InnerPk,
 
             // Cached
             hpk: [h_length]u8, // H(pk)
 
-            pub const packed_length: usize = InnerPk.packed_length;
+            /// Size of a serialized representation of the key, in bytes.
+            pub const compressed_length = InnerPk.compressed_length;
```

Comment:
For ECC, we have `compressed` and `uncompressed` coordinates, but indeed, in the context of Kyber, it may be confusing.

Not a big fan of `packed`. Would `serialized` work for you?

Reply:
Yes. Or `bytes_length`, `marshalled_length`, `wire_length`.

Reply:
We have quite a lot of `bytes_length` elsewhere, so let's use that!

Reply:
Done.